src/components/RestaurantList.jsx

```diff
@@ -1,79 +1,22 @@
-                            <img src='./category-chinese.png' alt='중식' className='category-icon' />
-                        </div>
-                        <div className='restaurant__info'>
-                            <h3 className='restaurant__name text-subtitle'>친친</h3>
-                            <p className='restaurant__description text-body'>
-                                Since 2004 편리한 교통과 주차, 그리고 관록만큼 깊은 맛과 정성으로 정통 중식의 세계를 펼쳐갑니다
-                            </p>
-                        </div>
-                    </li>
-
-                    <li className='restaurant'>
-                        <div className='restaurant__category'>
-                            <img src='./category-japanese.png' alt='일식' className='category-icon' />
-                        </div>
-                        <div className='restaurant__info'>
-                            <h3 className='restaurant__name text-subtitle'>잇쇼우</h3>
-                            <p className='restaurant__description text-body'>
-                                잇쇼우는 정통 자가제면 사누끼 우동이 대표메뉴입니다. 기술은 정성을 이길 수 없다는 신념으로 모든 음식에 최선을 다하는 잇쇼우는
-                                고객 한분 한분께 최선을 다하겠습니다
-                            </p>
-                        </div>
-                    </li>
-
-                    <li className='restaurant'>
-                        <div className='restaurant__category'>
-                            <img src='./category-western.png' alt='양식' className='category-icon' />
-                        </div>
-                        <div className='restaurant__info'>
-                            <h3 className='restaurant__name text-subtitle'>이태리키친</h3>
-                            <p className='restaurant__description text-body'>늘 변화를 추구하는 이태리키친입니다.</p>
-                        </div>
-                    </li>
-
-                    <li className='restaurant'>
-                        <div className='restaurant__category'>
-                            <img src='./category-asian.png' alt='아시안' className='category-icon' />
-                        </div>
-                        <div className='restaurant__info'>
-                            <h3 className='restaurant__name text-subtitle'>호아빈 삼성점</h3>
-                            <p className='restaurant__description text-body'>푸짐한 양에 국물이 일품인 쌀국수</p>
-                        </div>
-                    </li>
-
-                    <li className='restaurant'>
-                        <div className='restaurant__category'>
-                            <img src='./category-etc.png' alt='기타' className='category-icon' />
-                        </div>
-                        <div className='restaurant__info'>
-                            <h3 className='restaurant__name text-subtitle'>도스타코스 선릉점</h3>
-                            <p className='restaurant__description text-body'>멕시칸 캐주얼 그릴</p>
-                        </div>
-                    </li>
+                    {restaurants.map((restaurant) => (
```

Comment:
```suggestion
                    {restaurants.map(({ id, category, name, description}) => (
```
구조 분해 할당 문법으로 받으면 바로 접근할 수 있어서 좋은 것 같은데 세오의 생각은 어떠신가요??

Reply:
현재 5단계까지 진행을 한 상태에서 리팩토링을 진행하려다보니 이 부분에서 약간의 고민(?)이 생겼는데요.
구조 분해 할당을 하면 바로 접근이 가능하다는 이점이 있어서 너무 좋지만, 현재 <li> 태그를 클릭했을 때 선택된 resutaurant의 정보를 담아야 하는 상황이여 구조 분해 할당을 하면 선택된 정보를 담는 것이 어렵다는 생각이 들었습니다.
그래서 저는 구조 분해 할당을 하지 않고 resutaurant으로 하는 것이 좋다고 생각을 하는데 써밋은 이런 상황에서 어떤 선택을 하는게 더 좋다고 생각하시나요?🤔

Reply:
아마 주렁 코멘트 내용이랑 비슷한 것 같은데 저는 `resutaurant`도 사용하고 구조 분해 할당도 따로 사용해서 둘 다 사용할 수 있게 하는 것 같아요🙃
https://github.com/woowacourse/self-paced-react/pull/32#discussion_r2037695209